### DISCUSSION TITLE
ESAPI 2.5.5.0 release announcement

### DISCUSSION THREAD

**Discussion by kwwall** [ESAPI/esapi-java-legacy#857] [Announcements]
I undoubtedly confused a lot of people (including myself) by posting notifications for ESAPI 2.5.5.0 release that I was working on under the [ESAPI 2.5.4.0 release announcement](https://github.com/ESAPI/esapi-java-legacy/discussions/841).  I'm not  going to bother to try to clean those up (although I may delete them later), but did want to let you know that the ESAPI 2.5.5.0 release--after several unanticipated difficulties--is finally available in Maven Central.

I will finish up the release in GitHub with the formal release, including proper release announcement, later today.

In the meantime, if you want, you can find the release notes at:
- https://github.com/ESAPI/esapi-java-legacy/blob/develop/documentation/esapi4java-core-2.5.5.0-release-notes.txt

**Comment by kwwall** [comment]
It's soup. See https://github.com/ESAPI/esapi-java-legacy/releases/tag/esapi-2.5.5.0 for details.

**Reply by sh4nk** [reply]
@kwwall  does this release include solution for vulnerability : WS-2023-0429
https://github.com/advisories/GHSA-r68h-jhhj-9jvm

**Comment by kwwall** [comment]
Nope. Next one will. One year is not up until  11/24/2024.

-kevin
